UM/Settings/Models/InstanceContainersModel.py

```diff
@@ -117,6 +117,8 @@ def _fetchInstanceContainers(self) -> Tuple[Dict[str, InstanceContainer], Dict[s
                         containers[metadata["id"]] = registry.findContainers(id = metadata["id"])[0]
                     else:
                         metadatas[metadata["id"]] = metadata
+        if not containers and not metadatas:
+            print("Filter of {class_name} returned nothing! Filters are: {filters}".format(class_name = self.__class__.__name__, filters = self._filter_dicts))
```

Comment:
print()?
Also, while I nitpick a work in progress, 'data' is already a plural form.

Reply:
print() was an accident ^^